analyze_inspections.py

```diff
@@ -12,7 +12,10 @@
 import xml.etree.ElementTree as ET
 
 FILE_EXTENSIONS_TO_CONSIDER = [".kt", ".java", ".kts"]
-
+regexes = os.environ.get("INPUT_IGNORE_FILE_PATTERN","NoRegexSpecified")
+regexes_list = json.loads(regexes)
+regexes_combined = "|"
+regexes_combined = regexes_combined.join(regexes_list)
```

Comment:
Are you sure this does what it should? Isn't it safer to iterate through the regexes one at a time and match them individually?

Reply:
Currently it tries to match any pattern as specified [here](https://github.com/gps/intellij-inspections/pull/5/files#diff-9766a9038bb30fcb71f4e5455b50411033d1a730b4c79f3d4fa5ef4483b7077fR80) . To demonstrate this, i have added an additional `;` in `Database.kt` file which is ignored as shown [here](https://github.com/Resolut-Tech/BCN-WS/pull/128/checks?check_run_id=2268322700#step:10:6937)  .Other approach would be to write a helper function to iterate over each pattern and if any one matches then ignore results. Please let me know if changes are needed here.

Reply:
I would go with the helper function route.